source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -28,5 +27,90 @@ more naturally readable.
 Operations
 ----------
 
+The following sections provide information and examples for
+aggregation expression operations available in the driver.
+The operations are categorized by purpose and functionality.
+
+.. important::
+
+   The driver generates a Query API expression that may be different
+   from the Query API expression provided in each example. However,
+   the expressions are equivalent when evaluated.
+
+Boolean Expressions
+~~~~~~~~~~~~~~~~~~~
+
+You can construct a boolean expression from values of type
+``MqlBoolean`` and boolean aggregation operations. These 
+expressions evaluate to ``true`` or ``false``.
+
+The following table describes boolean aggregation expression operators
+available in the driver and equivalent expressions in the
+Query API. The methods link to API documentation and the Query API
+expressions link to Server manual documentation.
+
+.. note::
+
+   Some aggregation methods share a name with other methods available in
+   the driver. Check the parameter types and the type of
+   the object that the method is called on to confirm that that you are
+   referencing the correct documentation.
```

Comment:
The other methods (like the `and` [method](https://mongodb.github.io/mongo-java-driver/4.9/apidocs/mongodb-driver-core/com/mongodb/client/model/Filters.html#and(org.bson.conversions.Bson...))) cannot be called on an object, since they are static. (Also, this seems more general than just Booleans.)

I don't think that the issue is that users will end up looking at the wrong docs, but rather that they might read our example below (`and()`), think "I know that method" without clicking the link, and then try to use it. Perhaps we need an initial section that discusses how this integrates with the existing API, and this can be more fittingly mentioned there? Very roughly:

> To use all of the operators below, look for a method that takes a `TExpression`, like `Filters.expr` or `Projections.computed`. You can then start by obtaining the current document using `current()`. This API is typed, so you will need to specify the type of your field via, for example, `current().getInteger("myFieldName")`. [You can use IntelliSense to show which operations are available; maybe screenshot of code completion via `.`]. Values can also be specified via methods like `of`. While this API can be used within [existing] aggregation methods that take `TExpression`, various [existing] methods that return Bson values are not accepted by this API.

(The last sentence is intended to cover what I think is the original point.)

Reply:
Will discuss this point with my team to figure out how to incorporate this content into the intro. In the meantime, I have removed this admonition.